packages/templates/introspection/src/config.ts

```diff
@@ -0,0 +1,22 @@
+import { EInputType, GeneratorConfig } from 'graphql-codegen-core';
+import * as index from './template.handlebars';
+import { toJSON } from './helpers/to-json';
+
+export const config: GeneratorConfig = {
+  inputType: EInputType.SINGLE_FILE,
+  flattenTypes: true,
+  templates: {
+    index
+  },
+  primitives: {
```

Comment:
TypeScript says it's necessary to define them but I don't think this template even use it

Reply:
Primitives? it uses them in the template helpers